### PR TITLE
feat: add a `call` method and command

### LINKED ISSUE #10: gcx.call
A dev should be able to call a function from gcx.

Why? A HTTP trigger URL doesn't give errors whereas functions.call does.

```ts
const {call} = require('gcx');

async function main() {
  const {result, error} = await call({
    name: 'my-fn-name',
  });
  if (error) return console.error(error);
  console.log(result);
}
main();
```

See: https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions/call

### PR DESCRIPTION
**Pull request by grant**
Adds a call method for synchronously invoking a Cloud Function.
Fixes #10.

I usually don't like to just duplicate a PR, but I don't think I was in a good state with git, and I may have made a mistake with calling `super`.

### CODE REVIEW

File: test/test.ts [hunk 1]
```diff
@@ -151,3 +160,16 @@ function mockExists() {
       .get('/v1/projects/el-gato/locations/us-central1/functions/%F0%9F%A6%84')
       .reply(200);
 }
+
+/**
+ * @see https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions/call
+ */
+function mockCall() {
+  return nock('https://cloudfunctions.googleapis.com')
+      .post('/v1/*')
```
Comment (JustinBeckwith) [inline test/test.ts]: this should be `.post('/v1/%F0%9F%A6%84:call')`

File: test/test.ts [hunk 2]
```diff
@@ -107,13 +107,22 @@ describe('cloud functions api', () => {
...
     const scopes = [mockUploadUrl(), mockUpload(), mockDeploy(), mockPoll()];
     const projectId = 'el-gato';
     const deployer = new gcxp.Deployer({name, targetDir, projectId});
     await deployer.deploy();
     scopes.forEach(s => s.done());
   });
+
+  it('should call end to end', async () => {
+    const scopes =
+        [mockUploadUrl(), mockUpload(), mockDeploy(), mockPoll(), mockCall()];
```
Comment (JustinBeckwith) [inline test/test.ts]: You don't need all these mocks for call :)  Should just be `const scopes = [mockCall()];`

Comment (grant) [reply test/test.ts]: In fact, the other mocks seem to break the test.

File: test/test.ts (hunk 1 above)
Comment (grant) [reply test/test.ts]: Fixed. Thanks.

**Comment by JustinBeckwith** [comment]
Thank you again for adding this 🎉  We can come around and add the CLI part as needed in another PR.

**Pull request merged**